pkg/trace/api/otlp.go

```diff
@@ -341,10 +345,7 @@ func (o *OTLPReceiver) ReceiveResourceSpans(rspans ptrace.ResourceSpans, header
 	default:
 		log.Warn("Payload in channel full. Dropped 1 payload.")
 	}
-	return OTLPIngestSummary{
-		Hostname: hostname,
-		Tags:     attributes.RunningTagsFromAttributes(attr),
-	}
+	return src
```

Comment:
This change presents a small issue on this part of the exporter: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/9693/files#diff-9ca998552699e406631ae1a165fef81d5373ba6876136c8c48c10ae0355edf2aR125-R127

```go
switch src.Kind {
case source.KindHostname:
    hosts[src.Identifier] = struct{}{}
default:
    // What here? We don't have "attr" from attributes.RunningTagsFromAttributes(attr)
    // Is it rspan.Resource.Attributes() ?
}
```

Does it make sense to start pairing these types of changes with changes to https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/9693 ? Perhaps as PRs on top of that branch? WDYT?